### DISCUSSION TITLE
Plugin stopped working on Android

### DISCUSSION THREAD

**Discussion by boengli** [admob-plus/admob-plus#463] [Q&A]
I use this plugin for a while and everything worked perfectly fine for me.

**Simple implementation:**
```
this.platform.ready().then(async () => {
       await this.admob.start().then(() => {
            console.log('admob started');
       })

       const rewarded = new this.admob.RewardedAd({
            adUnitId: '<id>',
       })

      await rewarded.load()
      await rewarded.show()
    });
});
```
On Android "console.log('admob started');" isn't happening anymore.

**From AdMob.java:**
```
Log.i(TAG, "Initializing...");  // **This is called**
MobileAds.initialize(cordova.getActivity(), status -> {
      Log.i(TAG, "Initialized"); // **This is not! since 9th Nov.**
      helper.configForTestLab();
      callbackContext.success(new JSONObject(new HashMap<String, Object>() {{
           // looks like this is never called since 9th Nov.
      }}));
```

No error messages in console, just...
```
2021-11-25 16:36:11.078 21813-21916/com.xy.app I/AdMobPlus: Initialize plugin
2021-11-25 16:36:11.084 21813-21916/com.xy.app D/AdMobPlus: Execute ready
2021-11-25 16:36:33.437 21813-21916/com.xy.app D/AdMobPlus: Execute start
2021-11-25 16:40:59.846 21995-22094/com.xy.app I/AdMobPlus: Initializing...
```

On iOS everything works fine!
But it stopped working on Android for RewardedAd (Around Nov. 9th), the implementation or configuration hasn't changed.

Any ideas?

Thank you!

PS: Already tried upgrading or downgrading "admob-plus-cordova" as well as "play-services-ads", makes no difference.

Also:
https://stackoverflow.com/questions/70005348/admob-plus-cordova-stopped-working-on-android

Could be related to:
https://groups.google.com/g/google-admob-ads-sdk/c/zMshI4tmAwc

**Comment by boengli** [comment]
Doesn't anyone really have any ideas?

**Comment by kreso22** [comment]
I can't really help you @boengli but it sounds like you are hitting a dead end.

I can confirm however that the Android version does work for me. 

Your code looks OK to me. The only obvious difference from my code is that I never call `admob.start()`, I think the docs say it is automatically called.

I am guessing you tried everything already (like cleaning project, removing platforms, readding them etc..).

**Comment by boengli** [comment]
The last answer I got from the Google SDK Team was:

"_We've received the same issue reported by other publishers as well, and the team is now investigating this. We'll let you know the soonest we have our findings._"

**Comment by adri-igoox** [comment]
Try downgrading to v1.20.1, this has worked for me

**Reply by keev-studio** [reply]
how I can downgrade?

**Reply by adri-igoox** [reply]
In the config.xml `<plugin name="admob-plus-cordova" source="npm" spec="1.20.1">` or `cordova plugin add admob-plus-cordova@1.20.1`
